### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by romanlutz**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy
```
Comment (riedgar-ms) [inline requirements.txt]: Not now, but we need to discuss version locking

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- master
+- dev
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.5'
```
Comment (riedgar-ms) [inline azure-pipelines.yml]: Might need to consider other python versions, but this is fine for now

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+trigger:
+- master
```
Comment (riedgar-ms) [inline azure-pipelines.yml]: I like doing this on master, but surely the pytest tests don't exist there yet.....

File: requirements.txt (hunk 1 above)
Comment (romanlutz) [reply requirements.txt]: definitely!

File: azure-pipelines.yml (hunk 2 above)
Comment (romanlutz) [reply azure-pipelines.yml]: Yes, it's on my radar, but not my top priority

File: azure-pipelines.yml (hunk 3 above)
Comment (romanlutz) [reply azure-pipelines.yml]: next time we merge they will :-)

Comment (romanlutz) [reply azure-pipelines.yml]: Should I remove it?

Comment (romanlutz) [reply azure-pipelines.yml]: Actually since this is only in the dev  branch it won't affect master until we check this into master.

**Pull request merged**